### PR TITLE
OpenVino plugin reduce logical and, or

### PR DESCRIPTION
**Pull request by 1sanz**

### CODE REVIEW

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+using namespace plaidml;          // NOLINT[build/namespaces]
+using namespace InferenceEngine;  // NOLINT[build/namespaces]
+
+namespace PlaidMLPlugin {
+
+static OpRegistration reg("reducelogicalor", [](const Context& ctx) {
+  IE_ASSERT(ctx.operands.size() == 2);
+  auto I = ctx.operands.at(0);
+  std::vector<size_t> axes = get_axis_vector_from_constant_operand(1, ctx.layer);
+  auto* layer = dynamic_cast<ngraph::opset1::ReduceLogicalAnd*>(ctx.layer);
```
Comment (flaub) [inline plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp:22]: Is this right? Should it be `ReduceLogicalOr`?

**Comment by tzerrell** [comment]
Note: These are untested, as the OpenVINO test infrastructure in 2020.2 does not allow tests with boolean inputs and nGraph throws if these ops are given non-boolean inputs.

File: plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp (hunk 1 above)
Comment (tzerrell) [reply plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp:22]: It should be `ReduceLogicalOr`

Comment (cnamrata15) [reply plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp:22]: Oops, my bad.

Comment (1sanz) [reply plaidml/bridge/openvino/src/plaidml_plugin/ops/reduce_logical_or.cpp:22]: thanks @flaub I'll put in a PR to fix this

**Pull request merged**